content/resources/dutch-labour-law-workplace-action/index.en.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Protections for workplace action in Dutch Labour Law"
+date: "2025-03-19"
+lastmod: "2025-03-19"
+description: "An overview of the protections that Dutch Labour Law provides workers in relation to various workplace actions."
+tags: []
+layout: "single-resource"
+feature: "feature-action.jpg"
+---
+
+> **Disclaimer:** This document deals with legal issues and is intended to provide an informational resource only. Tech Workers Coalition doesn't in any way provide legal advice, so don't rely on this document as though it would constitute legal advice.
```

Comment:
This is great! I might even make it a shortcode so we can just keep a global version of it - I imagine we want it every now and then.